### PR TITLE
fix layer_norm half precision

### PR DESCRIPTION
**Pull request by tanyokwok**

### CODE REVIEW

File: examples/PyTorch/Inference/CUDA/BERT/main.py [hunk 1]
```diff
@@ -263,17 +263,17 @@ def run():
...
-        evaluate_torch(torch.compile(bert_large_amp, backend="disc"), inputs)
+        evaluate_torch(torch.compile(bert_large_amp, backend="inductor"), inputs)
 
-    # Run BladeDISC optimization.
-    print("BladeDISC Optimization.")
-    disc_optimize(bert_large_amp, inputs, 'bert_large_amp.disc.pt')
-    model = torch.jit.load('bert_large_amp.disc.pt').cuda().eval()
-    evaluate_torch(model, inputs)
+    # # Run BladeDISC optimization.
+    # print("BladeDISC Optimization.")
```
Comment (qiuxiafei) [inline examples/PyTorch/Inference/CUDA/BERT/main.py:273]: revert the comments?

File: pytorch_blade/tests/disc/ops/test_layer_norm.py [hunk 2]
```diff
@@ -44,5 +44,23 @@ def layernorm(x):
...
+            output = torch.layer_norm(input, normalized_shape, weight, bias, eps)
+            return output
+
+        width = 2560
+        input = torch.rand(1, 5, width, device="cuda", dtype=torch.half) * 0.1
+        weight = torch.ones(width, device="cuda", dtype=torch.half)
+        bias = torch.zeros(width, device="cuda", dtype=torch.half)
+
+        annotations = [([-1, -1, -1], torch.half), ([-1], torch.half), ([-1], torch.half)]
+        self._test_disc(layernorm, annotations, (input, weight, bias))
```
Comment (qiuxiafei) [inline pytorch_blade/tests/disc/ops/test_layer_norm.py:62]: nit: will this test pass fail on accuracy without converting input to fp32?
